src/Features/TestUtilities/Microsoft.CodeAnalysis.Features.Test.Utilities.csproj

```diff
@@ -1,10 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
```

Comment:
Why this change?

Reply:
This project was actually not added to the solution. So this is effectively a full file add. I brought this up to date with the equivalent EditorFeatures version.  Good possibly change that value though if not desired.